packages/components/src/components/HeroGradient.tsx

```diff
@@ -35,7 +35,14 @@ export const HeroGradient: React.FC<IHeroGradientProps> = ({
           <p {...restProps.descriptionProps}>{description}</p>
         </Info>
         <CTA>
-          {link && <a {...link} {...restProps.linkProps} />}
+          {link &&
+            toArray(link).map((link) => (
+              <a
+                key={`${link.href}${link.title}`}
```

Comment:
Title is optional? may be this is better
```suggestion
                key={`${link.href}${link.children}`}
```

Reply:
according to the def they are not optional:

```ts
export interface ILink {
  title: string;
  href: string;
  children?: string | React.ReactNode;
  rel?: string;
  target?: string;
  active?: boolean;
  onClick?: React.MouseEventHandler;
}
```

Reply:
I think this interface can not be truthy, because in 99% we provide children for links